File: 백준/Bronze/4153. 직각삼각형/직각삼각형.cc
#include <iostream>
using namespace std;

int main() {
  int a, b, c;

  for (;;) {
    cin >> a >> b >> c;

    if (a == 0 && b == 0 && c == 0)
      break;

    if (a > b && a > c) {
      if (a * a == b * b + c * c)
        cout << "right" << "\n";
      else
        cout << "wrong" << "\n";
    }

    if (b > a && b > c) {
      if (b * b == a * a + c * c)
        cout << "right" << "\n";
      else
        cout << "wrong" << "\n";
    }

    if (c > a && c > b) {
      if (c * c == a * a + b * b)
        cout << "right" << "\n";
      else
        cout << "wrong" << "\n";
    }
  }
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while True:
  s = list(map(int, input().split()))
  s.sort()

  a, b, c = s[0], s[1], s[2]
  if a == 0 and b == 0 and c == 0:
    break

  if c**2 == a**2 + b**2:
    print("right")
  else:
    print("wrong")